### PR TITLE
Adding repo docs

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Docs added

### Related Issues

- https://github.com/threefoldtech/tfgrid_stats/issues/58
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/191
- https://github.com/threefoldtech/tfgrid_stats/issues/61

### Checklist
- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [ ] Code format and docstrings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,40 @@
-# tfgrid-stats
+# Grid Stats
 
-to replace explorer.threefold.io with a simple page showing a capacity map + statistics
+![Version: 1.0.0](https://img.shields.io/github/v/release/threefoldtech/tfgrid_stats)
+[![CI-CD](https://github.com/threefoldtech/tfgrid_stats/actions/workflows/docker.yaml/badge.svg)](https://github.com/threefoldtech/tfgrid_stats/actions/workflows/docker.yaml)
+
+## Introduction
+
+Grid Stats is an application created to replace explorer.threefold.io with a simple page showing a capacity map + statistics.
```
Comment (maayarosama) [inline README.md]: is a web application

File: README.md [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
+
```
Comment (maayarosama) [inline README.md:9]: mention what the the capacity map and statistics exactly show

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Releasing process
+
+To create a new release there are some steps is required:
+
+## 1. Update Charts file
+
+In [charts.yaml](../tfgrid-stats/Chart.yaml) replace the value of `appVersion` with the new release tag.
```
Comment (maayarosama) [inline docs/release.md:7]: and we should replace the tag value in values.yaml

File: docs/release.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## 2. **Create a new release**
+
+  create a new release with same value of `appVersion` in `charts.yaml`.
+  
+  > You can find more details about creating a new release in [Release projects](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release).
+  
+## 3. **Create a new issue on [tf_operations](https://github.com/threefoldtech/tf_operations)**
+
+  Create a new issue of type `Update Request` and mention the following:
```
Comment (maayarosama) [inline docs/release.md:36]: mention creating a new tag while creating a new release

**Comment by maayarosama** [comment]
Open an issue for the lack of tests and code coverage

**Comment by maayarosama** [comment]
Add a small section giving a brief explanation for each available workflow

File: README.md (hunk 1 above)
Comment (zaelgohary) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (zaelgohary) [reply README.md:9]: done.

File: docs/release.md (hunk 3 above)
Comment (zaelgohary) [reply docs/release.md:7]: done.

File: docs/release.md (hunk 4 above)
Comment (zaelgohary) [reply docs/release.md:36]: done.

File: README.md [hunk 5]
```diff
@@ -1,23 +1,40 @@
-# tfgrid-stats
+# Grid Stats
 
-to replace explorer.threefold.io with a simple page showing a capacity map + statistics
+![Version: 1.0.0](https://img.shields.io/github/v/release/threefoldtech/tfgrid_stats)
+[![CI-CD](https://github.com/threefoldtech/tfgrid_stats/actions/workflows/docker.yaml/badge.svg)](https://github.com/threefoldtech/tfgrid_stats/actions/workflows/docker.yaml)
+
+## Introduction
+
+Grid Stats is a web application created to replace explorer.threefold.io with a simple page showing a capacity map + statistics. In the map, you get to see node distripution with each node location highlighted. While in Statistics section, general details about the nodes are being displayed like online nodes, farms count, total storage used, gateways ...etc.
```
Comment (maayarosama) [inline README.md]: node distribution

File: docs/release.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Releasing process
+
+To create a new release there are some steps is required:
+
+## 1. **Update Charts file**
+
+In [charts.yaml](../tfgrid-stats/Chart.yaml) replace the value of `appVersion` with the new release tag. Then, in [values.yaml](../tfgrid-stats/values.yaml) change the tag version with the new release tag.
```
Comment (maayarosama) [inline docs/release.md]: Separate the charts.yaml part and the values.yaml part and add ```tag:x.x.x``` as you did with the appVersion

File: docs/release.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+  ````yaml
+  appVersion: "x.x.x"
+  ````
+
+## 2. **Create a new release**
+
+  Create a new release with the new tag added in `appVersion` & `Charts.yaml`.
```
Comment (maayarosama) [inline docs/release.md:23]: Mention the steps with little description

File: docs/release.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+- **Release link**
+  
+    You can find it by changing the branch to the needed tag and then copy the URL.
+- **Charts directory link**
+  
+    In this case will be [tfgrid-stats](../tfgrid-stats/)
+
+  Newly created issue should look like this:
+  ![image](https://user-images.githubusercontent.com/62248851/210758782-1ff46374-9e4b-4e0d-a318-a7bf114022b1.png)
```
Comment (maayarosama) [inline docs/release.md]: change this screenshot to one updating tfgrid_stats rather than grid_weblets

File: docs/workflows.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
+# Weblets workflows
```
Comment (maayarosama) [inline docs/workflows.md]: Tfgrid_stats

File: docs/release.md (hunk 7 above)
Comment (zaelgohary) [reply docs/release.md:23]: All steps needed are provided in the link in the exact step.

File: README.md [hunk 10]
```diff
@@ -1,23 +1,40 @@
...
+  ```bash
+  git clone https://github.com/threefoldtech/tfgrid-stats.git
+  ```
 
 ## Get started
 
 Install the dependencies...
 
+Before running the Grid Stats, in your terminal move to the project directory, then install the required dependencies using
+
```
Comment (maayarosama) [inline README.md:27]: add 
``` cd tfgrid_stats```

**Comment by maayarosama** [comment]
LGTM, @xmonader please check.

File: docs/release.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
+# Releasing process
+
+To create a new release there are some steps is required:
+
+## 1. **Update Chart file**
+
+In [charts.yaml](../tfgrid-stats/Chart.yaml) replace the value of `appVersion` with the new release tag.
+
```
Comment (0oM4R) [inline docs/release.md:8]: maybe we need to add tags type, as in weblets
```suggestion

> NOTE: We use prereleases for Qanet and Testnet and releases for Mainnet.
### Tags type based on network:
| tag  | network | e.g
|-----|---------| ---------
|   --   | mainnet | 1.0.0
| qa   | qanet   | 1.0.0-qa1
| rc   | testnet | 1.0.0-rc1


```

File: README.md [hunk 12]
```diff
@@ -26,4 +44,8 @@ To create an optimised version of the app:
 npm run build
 ```
```
Comment (0oM4R) [inline README.md:45]: add how to serve it

**Review by 0oM4R — CHANGES_REQUESTED** [review]

File: docs/release.md (hunk 11 above)
Comment (zaelgohary) [reply docs/release.md:8]: No need to add tags for prereleases as the repo doesn't contain QA or Testnet.

File: README.md (hunk 12 above)
Comment (zaelgohary) [reply README.md:45]: done.

**Review by maayarosama — APPROVED** [review]

**Review by 0oM4R — APPROVED** [review]

**Pull request merged**